client/iface/bind/ice_bind.go

```diff
@@ -273,3 +289,15 @@ func fakeAddress(peerAddress *net.UDPAddr) (*net.UDPAddr, error) {
 	}
 	return newAddr, nil
 }
+
+func getMessages(msgsPool *sync.Pool) *[]ipv6.Message {
+	return msgsPool.Get().(*[]ipv6.Message)
```

Comment:
Reused the original code: https://github.com/netbirdio/wireguard-go/blob/6c340dd55aedca7e73c63266324cc75faa58416b/conn/bind_std.go#L222

I do not want to modify on it because it is easier to follow the changes with the upstream repo.

Reply:
// ipv6.Message and ipv4.Message are interchangeable as they are
// both aliases for x/net/internal/socket.Message.